exchanges/poloniex/poloniex_wrapper.go

```diff
@@ -1135,3 +1136,9 @@ func (p *Poloniex) GetServerTime(ctx context.Context, _ asset.Item) (time.Time,
 func (p *Poloniex) GetFuturesContractDetails(context.Context, asset.Item) ([]futures.Contract, error) {
 	return nil, common.ErrFunctionNotSupported
 }
+
+// GetLatestFundingRates returns the latest funding rates data
+func (p *Poloniex) GetLatestFundingRates(context.Context, *fundingrate.LatestRateRequest) ([]fundingrate.LatestRateResponse, error) {
+	// TODO: implement with API upgrade
+	return nil, common.ErrFunctionNotSupported
```

Comment:
I'm not really seeing the problem in that it isn't supported with the API versions where this is left. wrapper standards will also error on seeing a `NotYetImplemented` to prevent someone from just not implementing something that the API supports

Reply:
Ah fair enough if it errors it out then I am happy to resolve. Just wanted to make sure other tools will highlight pending functionality.

Reply:
Think I'll re-allow `NotYetImplemented` since I attacked most areas where it existed. I'll periodically disallow it to nit people unexpectedly 👀